scss/_pagination.scss

```diff
@@ -189,6 +189,14 @@
   @include media-breakpoint-between(md, lg) {
     @include pagination-max-items(8);
   }
+
+  @include media-breakpoint-between(lg, xl) {
+    @include pagination-max-items(10);
+  }
+
+  @include media-breakpoint-up(xl) {
+    @include pagination-max-items(12);
```

Comment:
Ok with the include.
But why 12 items? Just because it's 'previous breakpoint + 2'? I am currently wondering if it's not too much from an ergonomic point of view, even with a large screen?
And also, should we add a paragraph in the documentation to explain how to override these values?

Reply:
12 items because it seems to be the default design system item number [here](https://system.design.orange.com/0c1af118d/p/01f271-pagination/b/546660).
We should definitely have a paragraph explaining this behavior!, I'll try to tackle it asap.